guides/common/modules/con_troubleshooting-ruby.adoc

```diff
@@ -0,0 +1,19 @@
+[id="troubleshooting_ruby_{context}"]
+= Troubleshooting Ruby
+
+If the module fails to enable, it can mean an incorrect module is enabled.
+In that case, you have to resolve dependencies manually as follows.
+List the enabled modules:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# dnf module list --enabled
+----
+
+== Ruby
+
+If the Ruby 2.5 module has already been enabled, perform a module reset:
+[options="nowrap" subs="+quotes,attributes"]
+----
+# dnf module reset ruby
+----
```

Comment:
It doesn't show it, but when I checked it out locally it looks like there's no newline at the end of the file. We always include that.